kahuna/public/js/main.js

```diff
@@ -125,7 +125,7 @@ kahuna.run(['$log', '$rootScope', 'mediaApi', function($log, $rootScope, mediaAp
                     authAndRedirect(loginUriTemplate);
                 } else {
                     // Couldn't extract a login URI, die noisily
-                    throw error;
+                    throw new Error('Failed to redirect to auth, no login URI found');
```

Comment:
Note that we normally don't throw errors in Promise chains, we `$q.reject` instead. We should only throw if we want it to be surfaced noisily in our errors.